TestLoopX/src/testloop.cxx

```diff
@@ -1,9 +1,9 @@
 #include <cctk.h>
-#include "CarpetX/CarpetX/src/driver.hxx"
 #include <cctk_Arguments.h>
 #include <cctk_Parameters.h>
 #include <loop.hxx>
 #include <loop_device.hxx>
+#include <driver.hxx>
```

Comment:
I'd use the double quote form of include using `"` instead of hte `<>` form since none of these files are system include files.

See 

https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html

for the details (there won't be a difference in Cactus using gcc since we use `-I` to add the extra paths). 

The C99 standard say (in https://en.cppreference.com/w/cpp/preprocessor/include) that the double quote variant

> The intent of syntax (2) is to search for the files that are not controlled by the implementation.

ie include files not provided by the compiler.

Exactly where you put that cut-off between `"` and `<>` is a matter of some choice it seems.